### ISSUE TITLE
Custom sorting

### ISSUE THREAD

**Issue by mrjones2014** [mrjones2014/legendary.nvim#64]
API and how it works TBD.

This all depends on the resolution of this issue: https://github.com/stevearc/dressing.nvim/issues/22

**Comment by mrjones2014** [comment]
I think this is blocked by Telescope, not by Dressing :(

**Comment by mrjones2014** [comment]
Closing in favor of #218 which has more context
